webstart.ts

```diff
@@ -30,6 +30,41 @@ function assert_not_none(arg: any) : any {
   return arg;
 }
 
+function get_code_example(name: string) : string {
```

Comment:
Is there any way to store these in files and load them as part of the build process? This will be annoying to maintain. Look into webpack ways to import constant strings from files or similar